### PR TITLE
fix: issue templates

### PR DESCRIPTION
**Pull request by IlyesBenAmara**
# Motivation
Edited the issue template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Bug_report.yml [hunk 1]
```diff
@@ -2,15 +2,15 @@ name: "🐛 Bug Report"
 description: Create a new ticket for a bug, please specify its location (react|vue|core).
 title: "🐛 [BUG] - @eazychart-[core | react | vue]- <title>"
 labels: [
-  "bug"
+  "help-needed"
```
Comment (marrouchi) [inline .github/ISSUE_TEMPLATE/Bug_report.yml:5]: Why did you change the label ?

**Review by marrouchi — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/Bug_report.yml [hunk 2]
```diff
@@ -2,15 +2,15 @@ name: "🐛 Bug Report"
 description: Create a new ticket for a bug, please specify its location (react|vue|core).
 title: "🐛 [BUG] - @eazychart-[core | react | vue]- <title>"
```
Comment (marrouchi) [inline .github/ISSUE_TEMPLATE/Bug_report.yml:3]:
```suggestion
title: "🐛 [BUG] eazychart-[css | core | react | vue] - <title>"
```

Comment (marrouchi) [reply .github/ISSUE_TEMPLATE/Bug_report.yml:3]: Let's do the same for the others.

**Review by marrouchi — CHANGES_REQUESTED** [review]

**Comment by IlyesBenAmara** [comment]
Will close this PR in favor of #66 since this branch was not properly rebased

**Pull request closed without merging**